tests/testthat/_snaps/auditbayesianplanning/prior-and-expected-posterior-distribution-1.svg

```diff
@@ -29,8 +29,6 @@
-<circle cx='124.15' cy='481.54' r='5.12' style='stroke-width: 1.70; fill: #BEBEBE;' />
```

Comment:
These two points are now missing in all the prior-and-expected-posterior plots. Is that intended?